backend/substrapp/views/objective.py

```diff
@@ -246,6 +250,9 @@ def leaderboard(self, request, pk):
             })
         except LedgerError as e:
             return Response({'message': str(e.msg)}, status=e.status)
+        except Exception as e:
```

Comment:
if using one dedicated try/except, do you still need this?
If yes, using a dedicated method will be better for avoiding code repetition and keeping DRY.

Reply:
Updated !